diff -Naur serf-1.3.9/SConstruct serf-1.3.9.tpg/SConstruct
--- serf-1.3.9/SConstruct	2021-10-09 19:27:57.997244511 +0000
+++ serf-1.3.9.tpg/SConstruct	2021-10-09 19:29:24.956728859 +0000
@@ -315,7 +315,7 @@
 
   env.Append(LIBS=[apr_libs, apu_libs])
   if not env.get('SOURCE_LAYOUT', None):
-    env.Append(LIBPATH=['$APR/lib', '$APU/lib'],
+    env.Append(LIBPATH=['$APR/lib64', '$APU/lib64'],
                CPPPATH=['$APR/include/apr-1', '$APU/include/apr-1'])
   elif aprstatic:
     env.Append(LIBPATH=['$APR/LibR','$APU/LibR'],
@@ -328,7 +328,7 @@
   env.Append(LIBS=['zlib.lib'])
   if not env.get('SOURCE_LAYOUT', None):
     env.Append(CPPPATH=['$ZLIB/include'],
-               LIBPATH=['$ZLIB/lib'])
+               LIBPATH=['$ZLIB/lib64'])
   else:
     env.Append(CPPPATH=['$ZLIB'],
                LIBPATH=['$ZLIB'])
@@ -337,7 +337,7 @@
   env.Append(LIBS=['libeay32.lib', 'ssleay32.lib'])
   if not env.get('SOURCE_LAYOUT', None):
     env.Append(CPPPATH=['$OPENSSL/include/openssl'],
-               LIBPATH=['$OPENSSL/lib'])
+               LIBPATH=['$OPENSSL/lib64'])
   elif 0: # opensslstatic:
     env.Append(CPPPATH=['$OPENSSL/inc32'],
                LIBPATH=['$OPENSSL/out32'])
@@ -370,7 +370,7 @@
     apu_libs = ''
 
   env.Append(CPPPATH=['$OPENSSL/include'])
-  env.Append(LIBPATH=['$OPENSSL/lib'])
+  env.Append(LIBPATH=['$OPENSSL/lib64'])
 
 
 # If build with gssapi, get its information and define SERF_HAVE_GSSAPI
